components/permit-holders/AppHistoryCard.tsx

```diff
@@ -0,0 +1,111 @@
+import { Box, Link, Text, Wrap, Divider } from '@chakra-ui/react'; // Chakra UI
+import Table from '@components/internal/Table'; // Table component
+import RequestStatusBadge from '@components/internal/RequestStatusBadge'; // Request status badge component
+import PermitHolderInfoCard from '@components/internal/PermitHolderInfoCard'; // Custom Card Component
+
+// Placeholder data
+
+const DATA = Array(4).fill({
+  rcdPermitId: { rcdPermitId: 354 },
+  isRenewal: 'false',
+  requestStatus: 'EXPIRING',
+  expiryDate: '2021/01/01',
+  applicationId: { applicationId: 1 },
+});
+
+const COLUMNS = [
+  {
+    Header: 'Permit #',
+    accessor: 'rcdPermitId',
+    disableSortBy: true,
+    maxWidth: 140,
+    Cell: _renderPermitId,
+  },
+  {
+    Header: 'Request Type',
+    accessor: 'isRenewal',
+    disableSortBy: true,
+    maxWidth: 140,
+    Cell: _renderRequestType,
+  },
+  {
+    Header: 'Status',
+    accessor: 'requestStatus',
+    disableSortBy: true,
+    maxWidth: 190,
```

Comment:
I think the status column seems too narrow on my screen, and I'm using a MacBook Air which has a fairly standard screen size. Is there any way the column widths could be adjusted to give more space between the status column and expiry date?
![Screen Shot 2021-08-07 at 11 12 40 PM](https://user-images.githubusercontent.com/35577524/128620006-dbed2357-9da0-4c66-b33f-7944670d7a55.png)

Reply:
Agreed looks a bit too narrow on laptop screens, added 10px of padding to the right to address this
![image](https://user-images.githubusercontent.com/51224641/128628153-27feff00-d812-4b0e-9023-13708f50fe97.png)